source/modules/MEI.shared.xml

```diff
@@ -185,48 +185,23 @@
       </attDef>
     </attList>
   </classSpec>
-  <classSpec ident="att.ambitus.anl" module="MEI.shared" type="atts">
-    <desc xml:lang="en">Analytical domain attributes.</desc>
-    <classes>
-      <memberOf key="att.intervalHarmonic"/>
-    </classes>
-  </classSpec>
-  <classSpec ident="att.ambitus.ges" module="MEI.shared" type="atts">
-    <desc xml:lang="en">Gestural domain attributes.</desc>
-  </classSpec>
   <classSpec ident="att.ambitus.log" module="MEI.shared" type="atts">
     <desc xml:lang="en">Logical domain attributes.</desc>
   </classSpec>
-  <classSpec ident="att.ambitus.vis" module="MEI.shared" type="atts">
-    <desc xml:lang="en">Visual domain attributes.</desc>
-  </classSpec>
   <classSpec ident="att.ambNote.log" module="MEI.shared" type="atts">
     <desc xml:lang="en">Logical domain attributes.</desc>
     <classes>
       <memberOf key="att.accidental"/>
       <memberOf key="att.coloration"/>
-      <memberOf key="att.duration.logical"/>
+      <memberOf key="att.duration.log"/>
       <memberOf key="att.pitched"/>
     </classes>
   </classSpec>
-  <classSpec ident="att.anchoredText.anl" module="MEI.shared" type="atts">
```

Comment:
No anchoredText.log?

Reply:
I didn't introduce new attribute classes (iirc), only resolved problems found by the schematron rules…